packages/contracts/src/utils/JwtRegistry.sol

```diff
@@ -27,20 +27,19 @@ contract JwtRegistry is IDKIMRegistry, Ownable {
     }
 
     /// @notice Checks if a public key hash is valid and not revoked for a given kis and iss.
-    /// @param domainName The domain name contains kis, iss and azp fields.
+    /// @param domainName The domain name contains kid and iss fields.
     /// @param publicKeyHash The public key hash to validate.
     /// @return bool Returns true if the public key hash is valid and not revoked, false otherwise.
-    function isDKIMPublicKeyHashValid(
+    function isJwtPublicKeyHashValid(
         string memory domainName,
         bytes32 publicKeyHash
     ) public view returns (bool) {
         string[] memory parts = domainName.stringToArray();
-        string memory kidAndIss = string(abi.encode(parts[0], "|", parts[1]));
+        string memory kidAndIss = string(abi.encodePacked(parts[0], "|", parts[1]));
```

Comment:
@wshino Can you please change this to `IssAndKid` as the `kid` is a value which is related to the `iss`, just to name it correctly.

Reply:
@shreyas-londhe Does this change affect the order of the pubSignals?

Reply:
Yes, I will share the order with you.